src/core/symbology/qgsfillsymbollayer.h

```diff
@@ -1702,6 +1722,126 @@ class CORE_EXPORT QgsPointPatternFillSymbolLayer: public QgsImageFillSymbolLayer
                        double displacementX, double displacementY, double offsetX, double offsetY );
 };
 
+/**
+ * \ingroup core
+ * \class QgsRandomMarkerFillSymbolLayer
+ *
+ * A fill symbol layer which places markers at random locations within polygons.
+ *
+ * \since QGIS 3.12
+ */
+class CORE_EXPORT QgsRandomMarkerFillSymbolLayer : public QgsFillSymbolLayer
+{
+  public:
+
+    /**
+     * Constructor for QgsRandomMarkerFillSymbolLayer, with the specified \a pointCount.
+     *
+     * Optionally a specific random number \a seed can be used when generating points. A \a seed of 0 indicates that
+     * a truly random sequence should be used.
```

Comment:
Maybe it's worth being a bit more explicit here as to what 0 does. Something like: a seed of 0 indicates that a truly random sequence will be used on every rendering?